src/components/valences-test/ValencesTest.test.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import { Router } from "react-router-dom";
+import { render, screen, within, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { createMemoryHistory } from "history";
+
+import ElementManager from "../../ElementManager";
+import ValencesTest from "./ValencesTest";
+
+const history = createMemoryHistory();
+
+const setup = () => {
+  ElementManager.loadElements();
```

Comment:
already done in https://github.com/OpenSourceRaidGuild/atom-pwa/blob/next/src/setupTests.ts

also, the PR title `mock ElementManager` does not make much sense, this PR is to add a test for `ValencesTest`, no?

Reply:
Yeah, my bad. I opened this PR as a work in progress because I was struggling to mock `ElementManager` and needed some help, then I found a way to not mock it at all.